src/helpers/middlewares.ts

```diff
@@ -1,11 +1,19 @@
 import * as express from "express";
 import HttpStatusCodes from "http-status";
 import { getPerson, getBusiness } from "../db";
-import { MockPerson, MockBusiness, BusinessIdentification } from "./types";
+import {
+  MockPerson,
+  MockBusiness,
+  BusinessIdentification,
+  MockAccount,
+} from "./types";
 import generateID from "./id";
 
 export type RequestWithPerson = express.Request & { person?: MockPerson };
 export type RequestWithBusiness = express.Request & { business: MockBusiness };
+export type RequestWithBusinessAndAccount = RequestWithBusiness & {
```

Comment:
and this?

Reply:
fixed by https://github.com/kontist/mock-solaris/pull/210/commits/e3775b4c5303a92d70f12beb911efb2f45a22069